server/routes/nl.py

```diff
@@ -229,23 +239,29 @@ def _result_with_debug_info(data_dict: Dict, status: str,
       'counters': debug_counters,
       'data_spec': uttr_history,
   }
-  if query_detection.places_detected:
+
+  places_found_formatted = ""
+  for place in query_detection.places_detected.places_found:
+    places_found_formatted += f" (name: {place.name}, dcid: {place.dcid}); "
```

Comment:
nit: remove `;` there's `()` and space?

Reply:
i removed one extra space but kept the ; because in case we want to do a split later somewhere to show these nicely